pom.xml

```diff
@@ -81,12 +81,13 @@
         <ldaptive.version>1.2.3</ldaptive.version>
         <http.commons.version>4.5.13</http.commons.version>
         <cxf.version>3.4.5</cxf.version>
-        <kafka.version>2.5.0</kafka.version>
+        <kafka.version>2.8.1</kafka.version>
```

Comment:
Why are we not upgrading to the latest version `3.1.0`?

Reply:
need to test compatibility... 2.8.1 is what was suggested by whitesource...
and so spring-kafka is updated to 2.7.9